src/main/java/envoy/client/ui/listcell/MessageListCell.java

```
@@ -57,14 +58,13 @@ protected void updateItem(Message message, boolean empty) {
 			cell.getChildren().add(textLabel);
 			// Setting the message status icon and background color
 			if (message.getRecipientID() != client.getID()) {
-				final var statusIcon = new Label("", new ImageView(statusImages.get(message.getStatus())));
-				statusIcon.setPadding(new Insets(1, 0, 5, 5));
+				final var statusIcon = new ImageView(statusImages.get(message.getStatus()));
 				cell.getChildren().add(statusIcon);
 				cell.getStyleClass().add("own-message");
 			} else cell.getStyleClass().add("received-message");
 			// Adjusting height and weight of the cell to the corresponding ListView
 			cell.paddingProperty().setValue(new Insets(5, 20, 5, 20));
-			cell.prefWidthProperty().bind(listView.widthProperty().subtract(40));
+			if (listView != null) cell.prefWidthProperty().bind(listView.widthProperty().subtract(40));
```

Why should this fail silently? If the constructor is invoked with a `null` argument, I think an exception should be thrown instead.

---

See above